giotto/mapper/visualization.py

```diff
@@ -295,153 +172,216 @@ def create_network_3d(graph, data, node_pos, node_color, columns_to_color=None,
-                buttons=colorscale_buttons,
-                direction="down",
-                pad={"r": 10, "t": 10},
-                showactive=True,
-                x=0.12,
-                xanchor='left',
-                y=button_height,
-                yanchor="top"
-            ),
             go.layout.Updatemenu(
                 buttons=column_color_buttons,
                 direction="down",
                 pad={"r": 10, "t": 10},
                 showactive=True,
-                x=0.42,
+                x=0.11,
                 xanchor='left',
                 y=button_height,
                 yanchor="top"
             ),
         ])
 
-    fig.update_layout(
-        annotations=[
-            go.layout.Annotation(text="Colorscale:", x=0, xref="paper",
-                                 y=button_height - 0.03, yref="paper",
-                                 align="left", showarrow=False)
-        ],
-        autosize=False
-    )
-
-    if columns_to_color is not None:
+    if color_by_columns_dropdown:
         fig.add_annotation(
-            go.layout.Annotation(text="Color by:", x=0.37, xref="paper",
-                                 y=button_height - 0.03,
+            go.layout.Annotation(text="Color by:", x=0, xref="paper",
+                                 y=button_height - 0.045,
                                  yref="paper", align="left", showarrow=False)
         )
 
     return fig
 
 
-def create_interactive_network(pipeline, data, node_pos=None, node_color=None,
-                               columns_to_color=None, plotly_kwargs=None,
-                               summary_stat=np.mean):
+def create_interactive_network(pipeline, data, layout='kamada_kawai',
+                               layout_dim=2, color_variable=None,
+                               node_color_statistic=np.mean,
+                               color_by_columns_dropdown=True,
+                               plotly_kwargs=None):
     """
     Parameters
     ----------
-    pipeline : MapperPipeline
-        The nerve of the refined pullback cover. Nodes correspond to cluster
-        sets, and an edge exists between two nodes if they share at least one
-        point in common.
-
-    data : ndarray of shape (n_samples, n_features)
-        The point cloud data used to generate the nerve.
-
-    node_pos : igraph.Graph.layout or ndarray of shape (n_nodes, n_dims)
-        Encodes the layout of the graph according to a layout algorithm or
-        pre-defined array of coordinates in an n-dimensional space.
-
-    node_color : ndarray of shape (n_nodes,)
-        The numerical values to color each node of the graph by.
-
-    columns_to_color : dict, optional, default: ``None``
-        Key-value pairs (column_name, column_index) to specify which columns of
-        :attr:`data` to color the graph by. Generates a dropdown widget to
-        select the columns to color by.
+    pipeline : :class:`MapperPipeline` object
```

Comment:
Same comments on dosctring and default values as for static case

Reply:
Done